libs/designer-ui/src/lib/panel/panelcontainer.tsx

```diff
@@ -46,6 +46,8 @@ export type PanelContainerProps = {
   nodeId: string;
   title?: string;
   layerProps?: ILayerProps;
+  canResubmit?: boolean;
+  resubmitOperation?: () => void;
```

Comment:
Do we really need both of these? I am guessing one cannot exist without the other right? Can we just use "!!resubmitOperation" as the value for "canSubmit"?